spec/spec_helper.rb

```diff
@@ -35,6 +33,25 @@
   config.use_transactional_fixtures = true
   config.infer_spec_type_from_file_location!
   config.filter_rails_from_backtrace!
+
+  config.before :each, type: :system do
+    desired_capabilities = Selenium::WebDriver::Remote::Capabilities.chrome(
+      'chromeOptions' => {
+        'prefs' => {
+          'download.default_directory' => Rails.root.join('spec', 'downloads'),
+          'download.prompt_for_download' => false,
+          'plugins.plugins_disabled' => ['Chrome PDF Viewer']
+        }
+      }
+    )
+    driven_by :selenium,
+              using: :chrome,
+              options: { desired_capabilities: desired_capabilities }
+  end
+
+  config.before :suite do
```

Comment:
Opted for `.before :suite` rather than `.after :suite` in case you want to look at any of those downloads after a test run.